.eslintrc

```diff
@@ -81,96 +95,97 @@
-    "no-func-assign": 0,          // disallow overwriting functions written as function declarations
-    "no-inner-declarations": 0,   // disallow function or variable declarations in nested blocks
-    "no-invalid-regexp": 1,       // disallow invalid regular expression strings in the RegExp constructor
-    "no-irregular-whitespace": 0, // disallow irregular whitespace outside of strings and comments
-    "no-negated-in-lhs": 0,       // disallow negation of the left operand of an in expression
-    "no-obj-calls": 0,            // disallow the use of object properties of the global object (Math and JSON) as functions
-    "no-regex-spaces": 0,         // disallow multiple spaces in a regular expression literal
-    "no-reserved-keys": 0,        // disallow reserved words being used as object literal keys (off by default)
-    "no-sparse-arrays": 0,        // disallow sparse arrays
-    "no-unreachable": 0,          // disallow unreachable statements after a return, throw, continue, or break statement
-    "use-isnan": 0,               // disallow comparisons with the value NaN
-    "valid-jsdoc": 0,             // Ensure JSDoc comments are valid (off by default)
-    "valid-typeof": 0,            // Ensure that the results of typeof are compared against a valid string
+    "no-dupe-keys": 2,            // disallow duplicate keys when creating object literals
+    "no-empty": 2,                // disallow empty statements
+    "no-empty-character-class": 2,// disallow the use of empty character classes in regular expressions
+    "no-ex-assign": 2,            // disallow assigning to the exception in a catch block
+    "no-extra-boolean-cast": 1,   // disallow double-negation boolean casts in a boolean context
+    "no-extra-parens": 1,         // disallow unnecessary parentheses (off by default)
+    "no-extra-semi": 2,           // disallow unnecessary semicolons
+    "no-func-assign": 2,          // disallow overwriting functions written as function declarations
+    "no-inner-declarations": 1,   // disallow function or variable declarations in nested blocks
+    "no-invalid-regexp": 2,       // disallow invalid regular expression strings in the RegExp constructor
+    "no-irregular-whitespace": 2, // disallow irregular whitespace outside of strings and comments
+    "no-negated-in-lhs": 2,       // disallow negation of the left operand of an in expression
+    "no-obj-calls": 2,            // disallow the use of object properties of the global object (Math and JSON) as functions
+    "no-regex-spaces": 2,         // disallow multiple spaces in a regular expression literal
+    "no-sparse-arrays": 2,        // disallow sparse arrays
+    "no-unreachable": 2,          // disallow unreachable statements after a return, throw, continue, or break statement
+    "use-isnan": 2,               // disallow comparisons with the value NaN
+    "valid-jsdoc": 1,             // Ensure JSDoc comments are valid (off by default)
```

Comment:
Let's turn this off. Flow types should be plenty to document function arguments and returns, jsDoc is redundant.

Reply:
This just warns about invalid jsdocs, it doesn't require adding them

Reply:
I don't even think it should warn. It's just extra noise.

Reply:
should we just remove the existing jsdocs then?

Reply:
Go for it; I think there are some older files I wrote which used them. I gave up on them pretty quickly. We can tear out the `@param` parts of those but leave the general descriptions.